browser-test/src/support/applicant_questions.ts

```diff
@@ -960,16 +960,38 @@ export class ApplicantQuestions {
       'This question is required',
     )
   }
+  async expectErrorOnReviewModal(northStarEnabled = false) {
+    if (northStarEnabled) {
```

Comment:
**suggestion:** Thank you for adding all of these assertions, but I think we can assert the same text as non-northstar (i.e. don't have to add text assertions for everything). You could also consider creating variables for the text, like `modalTitle`, `modalContent`, and `modalButton` and then using that in the branches.

Reply:
Thanks for the suggestion.  refactored the method. PTAL.